tools/benchmark_tool/openvino/tools/benchmark/utils/utils.py

```diff
@@ -270,7 +307,35 @@ def get_command_line_arguments(argv):
         parameters.append((arg_name, arg_value))
     return parameters
 
-def parse_input_parameters(parameter_string, input_info):
+
+def get_input_output_names(nodes):
+    return [node.get_friendly_name() for node in nodes] # get_friendly_name() or get_name() ?
```

Comment:
Answering your question: I suppose `get_friendly_name` should be here

Reply:
may be it's better to use `friendly_name` property (same as `get_friendly_name` but more pythonic)